### ISSUE TITLE
Add endpoint for operational monitoring of server healt status

### ISSUE THREAD

**Issue by petrkalina** [dcm4che/dcm4chee-arc-light#1090]
We could have a single health http endpoint without authentication that returns a 200 or 500 status matching general status and also returns a json doc with details. 

Something like:

```
{
dicom: ok,
hl7: ok,
ui: ok,
db: ok,
storage space: warning 
}
```

**Comment by gunterze** [comment]
What's wrong with (still) using Nagios for that?

**Comment by petrkalina** [comment]
In case there is a load-balanced environment, it has to decide about the health of the archive services. It is possible to do this indirectly via nagios, but this adds another dependency between the load-balancer and the archive. This would enable a simple uniform check that the load-balancers could use for all the archive endpoints. And the nagios could use this too!

**Comment by isoinh** [comment]
Yes, would be pretty nice to be able to check all of the internally verified health statuses from one place and integrate that for LB and other external monitoring services.
And of course possibility to bypass authentication services to simplify things.
